src/com/palmergames/bukkit/towny/listeners/TownyPlayerListener.java

```diff
@@ -101,11 +102,19 @@ public TownyPlayerListener(Towny instance) {
 
 	@EventHandler(priority = EventPriority.NORMAL)
 	public void onPlayerJoin(PlayerJoinEvent event) {
-
+		TownyPermissionSource permissionSource = TownyUniverse.getInstance().getPermissionSource();
+		
 		Player player = event.getPlayer();
 
+		// Safe Mode Join Messages
 		if (plugin.isError()) {
-			TownyMessaging.sendMessage(player, Colors.Rose + "[Towny Error] Locked in Safe mode!");
+			// Operator or an admin.
+			if (player.isOp() || permissionSource.has(player, PermissionNodes.TOWNY_ADMIN.getNode())) {
+				TownyMessaging.sendMessage(player, Colors.Rose + "[Towny Error] Towny is locked in Safe Mode due to an error! Please check the server's console for more information.");
+				return;
+			}
+			// Regular player.
+			TownyMessaging.sendMessage(player, Colors.Rose + "[Towny Error] Towny is locked in Safe Mode due to an error! Tell an admin to check the server's console.");
```

Comment:
```suggestion
			String msg = "[Towny Error] Towny is locked in Safe Mode due to an error! Tell an admin to check the server's console.";
			// Operator or an admin.
			if (TownyUniverse.getInstance().getPermissionSource().isTownyAdmin(player)) {
				msg = "[Towny Error] Towny is locked in Safe Mode due to an error! Please check the server's console for more information.";
			}
			TownyMessaging.sendErrorMsg(player, msg);
```

Reply:
Having the error messages be translatable would also be a good idea